bundle/metadata/annotations.yaml

```diff
@@ -9,7 +9,7 @@ annotations:
   operators.operatorframework.io.metrics.builder: operator-sdk-v1.32.0+git
   operators.operatorframework.io.metrics.mediatype.v1: metrics+v1
   operators.operatorframework.io.metrics.project_layout: go.kubebuilder.io/v3
-  com.redhat.openshift.versions: "=v4.14"
+  com.redhat.openshift.versions: "=v4.16"
```

Comment:
Moving this to `v4.16` may be an issue with CVP.  I think that is why master was still at `v4.14`, (which should probably now be updated).  @mansikulkarni96 could you confirm my statement?

Reply:
Correct we can switch to 4.15 now and bump to 4.16 in CY24Q1